neurovault/apps/statmaps/views.py

```diff
@@ -246,7 +246,12 @@ def edit_image(request, pk):
     else:
         form = form(request.user, instance=image)
 
-    context = {"form": form}
+    # Retrieve cognitive atlas contrast selector autocomplete
+    django_field = "contrast_definition_cogatlas"
```

Comment:
This breaks for formsets where the field is "image_set-0-contrast_definition_cogatlas", "image_set-1-contrast_definition_cogatlas" etc.